tests/Main.fs

```diff
@@ -38,9 +38,17 @@ let ``Missing FablePackageType property should report an error`` () =
 
 [<Test>]
 let ``Missing FableTarget property should report an error`` () =
-    expectToFailWithMessage
-        Workspace.fixtures.invalid.``MissingFableTarget.fsproj``
-        "You need to set at least one of Fable target via the PackageTags property. Possible values are: fable-dart, fable-dotnet, fable-javascript, fable-python, fable-rust, fable-all."
+    task {
+        do!
+            expectToFailWithMessage
+                Workspace.fixtures.invalid.``MissingFableTarget.fsproj``
+                "You need to set at least one of Fable target via the PackageTags property. Possible values are: fable-dart, fable-dotnet, fable-javascript, fable-python, fable-rust, fable-all."
+
+        do!
+            expectToFailWithMessage
+                Workspace.fixtures.invalid.``MissingFableTargetMultiTFM.fsproj``
+                "You need to set at least one of Fable target via the PackageTags property. Possible values are: fable-dart, fable-dotnet, fable-javascript, fable-python, fable-rust, fable-all."
+    }
```

Comment:
Please avoid testing 2 thing in the same test. 

This make it difficult to know the exact source of the error

We can create another tests named:

```fs
let ``Missing FableTarget property should report an error - MultiTFM`` () =
```